app/assets/javascripts/discourse/app/lib/autocomplete.js

```diff
@@ -17,8 +17,9 @@ import discourseLater from "discourse-common/lib/later";
 
 export const SKIP = "skip";
 export const CANCELLED_STATUS = "__CANCELLED";
-const allowedLettersRegex = /[\s\t\[\{\(\/]/;
-let _autoCompletePopper;
+
+const ALLOWED_LETTERS_REGEXP = /[\s[{(/]/;
```

Comment:
The `\t` is already included in `\s`.
Also removed the `/` used to escape the characters which isn't required in a `[]`.